src/core/operations/ToModhex.mjs

```diff
@@ -0,0 +1,57 @@
+/**
+ * @author linuxgemini [ilteris@asenkron.com.tr]
+ * @copyright Crown Copyright 2020
+ * @license Apache-2.0
+ */
+
+import Operation from "../Operation.mjs";
+import { toModhex, TO_MODHEX_DELIM_OPTIONS } from "../lib/Modhex.mjs";
+import Utils from "../Utils.mjs";
+
+/**
+ * To Modhex operation
+ */
+class ToModhex extends Operation {
+
+    /**
+     * ToModhex constructor
+     */
+    constructor() {
+        super();
+
+        this.name = "To Modhex";
+        this.module = "Default";
+        this.description = "Converts the input string to modhex bytes separated by the specified delimiter.";
+        this.infoURL = "https://en.wikipedia.org/wiki/YubiKey#ModHex";
+        this.inputType = "ArrayBuffer";
+        this.outputType = "string";
+        this.args = [
+            {
+                name: "Delimiter",
+                type: "option",
+                value: TO_MODHEX_DELIM_OPTIONS
+            },
+            {
+                name: "Bytes per line",
+                type: "number",
+                value: 0
+            }
+        ];
+    }
+
+    /**
+     * @param {ArrayBuffer} input
+     * @param {Object[]} args
+     * @returns {string}
+     */
+    run(input, args) {
+        let comma;
```

Comment:
What's up with this variable? Why is it here?

Reply:
This is passed along to the `toHex` function indirectly, I think this can be removed.

Reply:
This is removed and the accompanying function call now has `undefined` on its place.